### ISSUE TITLE
About Validation

### ISSUE THREAD

**Issue by onetance** [Zhang-xiaokun/CoHHN#3]
Thanks for sharing this awesome project! I am very interested in your paper, and I would like to ask you a question.
Does the model that get the results recorded in the paper use the validation set for training? Are the hyperparameters obtained on the validation set first, and then the model is retrained on the training set and validation set? I can't seem to get the results in the paper.

**Comment by Zhang-xiaokun** [comment]
Thanks for your attention on our work~
We have released all source codes of our work.
You can employ the source codes and processed data to reproduce our results. Specifically, our training data is in "train.txt", and test data is in "test.txt". You can directly run the model to reproduce the results presented in our paper. We have determined the hyper-parameter using the validation set, so you just need to set the corresponding hyper-parameters according to our paper and run the model.
Also, we provided all pre-processing codes which you can use to easily process your own data.
